client/src/components/Admin/CourseUpload/CourseUpload.js

```diff
@@ -65,6 +67,7 @@ export default function CourseUpload() {
     } else {
       alert("Chapter added scuccessfully!");
```

Comment:
Instead of giving alert you can checkout react-toastify to add notifications. 
https://www.npmjs.com/package/react-toastify

Reply:
Will add it in the next PR.